main.cpp

```diff
@@ -34,6 +109,5 @@ int main() {
         cin.get();
```

Comment:
Taka ciekawostka: tutaj są `cin.get()`y, bo jeżeli będziesz coś wpisywał i akurat coś czymś rzuci, to wtedy jak coś wpiszesz po tym to program się nie zamknie bez powodu, tylko będzie czekał na enter... A poza tym wtedy, kiedy ta linia była pisana, nie mieliśmy jeszcze portable `getch()`y napisanych